plugins/steelseries/README.md

```diff
@@ -17,6 +17,7 @@ available from the vendor.
 This plugin supports the following protocol IDs:
 
 * com.steelseries.fizz
+* com.steelseries.fizz.tunnel
```

Comment:
do you think this really is a different protocol?

Reply:
Ya, not convinced personally.

Reply:
That is a question I wanted to ask you. your remark seems to tell that it is fair to not have a new protocol that proxy command to another.

Reply:
So I will drop it